appeals/web/src/server/lib/dates.js

```diff
@@ -115,3 +115,19 @@ export function webDateToDisplayDate(dayMonthYear, { condensed = false } = {}) {
 		return '';
 	}
 }
+
+/**
+ * @param {string | undefined} time24
+ * @returns {string | undefined}
+ */
+export function convert24hTo12hFormat(time24) {
```

Comment:
Could you add unit tests for this function? It's a fairly complex conversion so it would be good to check.

Reply:
Also I know the fact this operates on times is implied by the existing name, but it might be better to explicitly include `time` in the function name, just to avoid any potential doubt